### ISSUE TITLE
Roadmap to 1.0

### ISSUE THREAD

**Issue by bajtos** [dittach/loopback-connector-riak#1]
Hi, this is Miroslav from StrongLoop, a LoopBack core developer. I would like to include your connector in the list of connectors offered by `yo loopback:datasource` - see [loopback-workspace/available-connectors.json](https://github.com/strongloop/loopback-workspace/blob/master/available-connectors.json).

However, I am rather reluctant to add a connector that is in a version `0.0.x`.

Do you have any plans to move at least to `0.1.x`, or even better to `1.x.y`?

Quoting http://semver.org/:

> How do I know when to release 1.0.0?

> If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you're worrying a lot about backwards compatibility, you should probably already be 1.0.0.

**Comment by onyxrev** [comment]
Of course. Dittach hopes to ship our software with this connector into production soon. I guess it makes sense to christen 1.0.0 at the time. In the meantime the community is welcome to try it out. This software should probably be in a 0.1.x state version at the moment, if we were to follow semver's recommendation.

**Comment by onyxrev** [comment]
I'll bump to 0.1.0 when this passes the loopback-datasource-juggler tests.

**Comment by bajtos** [comment]
Sounds good! Once it's done, would you mind sending a pull request to loopback-workspace adding your connector to the list?

**Comment by onyxrev** [comment]
Can do.
